lib/phony/countries/myanmar.rb

```diff
@@ -0,0 +1,18 @@
+# Myanmar: http://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000920003PDFE.pdf
+# + formatting according to Google's libphonenumber library. (r680 7/30/2014)
+#
+# note that ANEE SAKHAN area, Yangon ZTE GSM, MEC Tel CDMA 800 Mhz is not
+# correctly handled by the google library
```

Comment:
sorry, my wording is indeed confusing. What I meant: If you were to try some of the myanmar numbers (e.g. ANEE SAKHAN area), Google's libphonenumber library will tell you that they are not valid, even though they are. The phony rules for myanmar does not replicate that aspect of the google library because it's wrong.